CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [2.2.0] - 2020-01-21
+### Removed
+
+* Support for ^1.0 releases of phphttp/httplug
```

Comment:
Convention seems for this sentence to start with `Dropped` so it's clearer just by reading this sentence that behaviour has been dropped rather than added.

Reply:
Further, to improve readability I might suggest something like this for the changelog record:

* Dropped support for ^1.0 releases of HTTPlug (phphttp/httplug)
    * HTTP clients must now use the `Psr\Http\Client\ClientInterface` rather than `Http\Client\HttpClient` interface.
    * If you are using HTTPlug you should upgrade to version ^2.0 of HTTPlug

Reply:
The ^2.0 release of HTTPlug has the HttpClient interface extend ClientInterface, giving full backward compatibility. No code changes should be needed *assuming* they're using ^2.0. If they're using ^1.0 this PR will break their app. The only reason they'd be using ^1.0 is if they're using sock-client < 2.0.0-beta1.

Reply:
Ah OK. Maybe the changelog message could mention something then about dropping support for sock-client < 2.0.0-beta1?

Is it just sock-client old versions that we are dropping support for here or could there be other http clients that a user is using that use the older interface?